atmega328/sonoffsc/sonoffsc.ino

```diff
@@ -39,9 +41,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define DHT_PIN                 6
 #ifndef DHT_TYPE
-#define DHT_TYPE                DHT11
+#define DHT_TYPE                DHT22
```

Comment:
I'm also using a DHT22 but I left it configured for the DHT11 since that's the sensor that comes with the SonoffSC